docs/architecture/blazor-for-web-forms-developers/data.md

```diff
@@ -1,22 +1,152 @@
 ---
 title: Data access and management
 description: Learn how to access and handle data in ASP.NET Web Forms and Blazor.
-author: danroth27
-ms.author: daroth
-ms.date: 09/11/2019
+author: csharpfritz    
+ms.author: jefritz
+ms.date: 26/04/2020
```

Comment:
```suggestion
ms.date: 04/26/2020
```
I believe the platform would incorrectly parse this date, instead it expects `MM/DD/YYYY` format.